Source/UltraleapTrackingCore/UltraleapTracking.Build.cs

```diff
@@ -213,8 +213,8 @@ public bool LoadLeapLib(ReadOnlyTargetRules Target)
 
 				AdditionalPropertiesForReceipt.Add("AndroidPlugin", Path.Combine(ModulePath, "UltraleapTracking_APL.xml"));
 
-                		PrivateDependencyModuleNames.AddRange(new string[] { "Launch" });
-            		}	
+                PrivateDependencyModuleNames.AddRange(new string[] { "Launch" });
```

Comment:
If the above uses `.Add` only for a single member then this should do the same for consistency sake